packages/client/src/forms/configuration/defaultUtils.ts

```diff
@@ -71,21 +71,30 @@ export function getDefaultField(
 
 export function configureDefaultQuestions(
   defaultFieldCustomisations: IDefaultFieldCustomisation[],
-  defaultEventForm: ISerializedForm
+  defaultEventForm: ISerializedForm,
+  inConfig: boolean
```

Comment:
the inConfig boolean allows us to splice disabled fields away from the form only when in registration, not configuration mode.